lib/sequent/configuration.rb

```diff
@@ -127,6 +128,7 @@ def initialize
       self.time_precision = DEFAULT_TIME_PRECISION
 
       self.enable_autoregistration = false
+      self.enable_offline_migration_read_only_mode = false
```

Comment:
I think this might actually be good default behaviour, since the alternative is that the application is not running or the projections would have missing data.

Reply:
Also, to be documented on release.